dll/cpl/desk/lang/en-US.rc

```diff
@@ -76,6 +76,7 @@ CAPTION "Advanced Appearance"
 FONT 8, "MS Shell Dlg"
 BEGIN
     CONTROL "", IDC_APPEARANCE_PREVIEW, "PreviewWndClass", WS_VISIBLE | WS_BORDER, 7, 7, 232, 120
+    LTEXT "If you select a windows and buttons setting other than ReactOS classic, it will override the following settings, except in some older programs.", IDC_ADVAPPEARANCE_DISCLAIMER, 7, 133, 232, 30, BS_MULTILINE
```

Comment:
```suggestion
    LTEXT "If you select a theme other than ReactOS classic, it will override the following settings, except in some older programs.", IDC_ADVAPPEARANCE_DISCLAIMER, 7, 133, 232, 30, BS_MULTILINE
```
(ditto for others)

Reply:
OK, change in progress.

Reply:
Commit & push done